Fix social media sharing

lantelyes opened the pull request:
#### What does this PR do?
Updates the sharing copy for twitter

#### Description of Task to be completed?
Fix the hashtag copy error when sharing via twitter on donation and pension pages

#### How should this be manually tested?
Go to the various donation/pension pages and click the twitter share button

#### What are the relevant Trello Card stories? [ Trello Card Item (https://trello.com/c/SIhNL5av/108-fix-sharing-hashtag-bug)


#### Pull Request Type (check one)
- [ ] Feature
- [x] Bug Fixes
- [ ] Chore

#### Developer Checklist
- [x] New code has been scanned for ESlint issues
- [ ] I have tested my code locally ( See note below )
- [x] I made sure that there are no other pull requests that solve for this issue
- [x] My code follows the same structure within the guidelines

Note:
This is WIP, there seems to be no data for movements and protestors in the donation page, and policies in the pension page. So I have not been able to test those

Easybuoy commented:
Tested, and it only works for Twitter, is there any plan for doing for the other social media platforms? @lantelyes

lantelyes commented:
I can add copy to the other platforms as well, should it be the same as the copy for twitter?

Easybuoy commented:
> I can add copy to the other platforms as well, should it be the same as the copy for twitter?

Yes, It should be

Easybuoy commented:
Also noticed the profile detail sharing hasn't been done yet as well

Easybuoy commented:
Good job, it looked great on Twitter

lantelyes commented:
Thanks! And thanks for the clarification. I'll be home in an hour or so, and update the other social platforms with this copy, and complete the profile detail sharing

lantelyes commented:
@Easybuoy I've updated all the links to use the same copy, and added the copy to the profile details page (I used the same copy as for victim donations)

Easybuoy commented:
Have you been able to test this? https://github.com/Say-Their-Name/say-their-names-web/issues/62#issuecomment-641431288

Easybuoy commented:
> @Easybuoy I've updated all the links to use the same copy, and added the copy to the profile details page (I used the same copy as for victim donations)

Looks good

ashleygraf101 commented:
Whatsapp doesn't have hashtags, but it could be a great visual point of interest in group chats.

lantelyes commented:
> Have you been able to test this? [#62 (comment)](https://github.com/Say-Their-Name/say-their-names-web/issues/62#issuecomment-641431288)

Not yet, @devingray I still don't see the data in the database, could you make sure it was published?

ashleygraf101 commented on src/pages/PetitionDetail.js:
```diff
@@ -13,6 +13,18 @@ import config from '../utils/config';
+    socialCopy = `Help bring justice to ${petitionDetails.person.full_name} by signing this petition and put pressure on the authorities to do the right thing. If we show up in numbers, we can win. 
+#SayTheirNames #BlackLivesMatter #${petitionDetails.person.full_name.replace(/\s+/g, '')}`;
+  } else {
+    socialCopy = 'Sign this petition to demand policy change and bring media attention to the number of people who want action. #BlackLivesMatter #SayTheirNames';
```
Perhaps change to 'bring attention to how many of us want action'?

ashleygraf101 commented on src/pages/DonationDetail.js:
```diff
@@ -13,6 +13,18 @@ import config from '../utils/config';
+    socialCopy = `Join me in donating to ${donationDetails.person.full_name}’s fund, and supporting their family’s fight for justice. 
+#SayTheirNames #BlackLivesMatter #${donationDetails.person.full_name.replace(/\s+/g, '')}`;
+  } else {
+    socialCopy = `The movement goes on. Join me in supporting ${donationDetails.fund} and let’s help fuel the campaign for justice together. #BlackLivesMatter #SayTheirNames`;
```
Remove 'help'. In retrospect qualifiers weaken the message.

ashleygraf101 commented on src/pages/DonationDetail.js:
```diff
@@ -13,6 +13,18 @@ import config from '../utils/config';
+const getSocialCopy = (donationDetails) => {
+  let socialCopy;
+  if (donationDetails.person) {
+    socialCopy = `Join me in donating to ${donationDetails.person.full_name}’s fund, and supporting their family’s fight for justice. 
```
Perhaps add 'We remember (full_name}' to the start, and then change from '{full_name}'s fund' to 'their memorial fund'.

lantelyes commented:
> I just made some copy suggestions. Code looks good!

Good suggestions! made the changes

ashleygraf101 commented on src/pages/DonationDetail.js:
```diff
@@ -13,6 +13,18 @@ import config from '../utils/config';
+const getSocialCopy = (donationDetails) => {
+  let socialCopy;
+  if (donationDetails.person) {
+    socialCopy = `'We remember ${donationDetails.person.full_name}. Join me in donating to thier memorial fund, and supporting their family’s fight for justice. 
```
```suggestion
    socialCopy = `'We remember ${donationDetails.person.full_name}. Join me in donating to their memorial fund, and supporting their family’s fight for justice. 
```

Easybuoy commented:
I asked Devin, he said he's pushed the changes, but if there're still issues he'd fix tomorrow. You can reach out to him on slack if they're still issues

Easybuoy commented:
I think we should merge this, we'd fix any other issue that comes up in another pull request.

This is MVP critical
